apps/country/mutations.py

```diff
@@ -75,7 +75,14 @@ class Arguments(ExportBaseMutation.Arguments):
     DOWNLOAD_TYPE = ExcelDownload.DOWNLOAD_TYPES.COUNTRY
 
 
+class ExportMonitoringSubRegion(ExportBaseMutation):
```

Comment:
Let's use plural for consistency.
```suggestion
class ExportMonitoringSubRegions(ExportBaseMutation):
```